dll/win32/comdlg32/lang/cdlg_Pt.rc

```diff
@@ -301,34 +301,34 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Procurar"
 FONT 8, "MS Shell Dlg"
 {
-    LTEXT "&Procurar:", -1, 4, 8, 52, 8 //FIXME: accelerator collision &P
+    LTEXT "&Procurar:", -1, 4, 8, 52, 8
     EDITTEXT edt1, 57, 7, 148, 12, ES_AUTOHSCROLL | WS_BORDER | WS_GROUP | WS_TABSTOP
     CHECKBOX "Palavra &inteira", chx1, 4, 26, 140, 12, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
     CHECKBOX "&Maiúsculas/minúsculas", chx2, 4, 42, 140, 12, BS_AUTOCHECKBOX | WS_TABSTOP
     GROUPBOX "Direcção", grp1, 147, 21, 58, 38
-    AUTORADIOBUTTON "&Acima", rad1, 151, 30, 48, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP//FIXME: accelerator collision &A
+    AUTORADIOBUTTON "&Acima", rad1, 151, 30, 48, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | WS_TABSTOP
```

Comment:
The `WS_VISIBLE` is redundant.

Reply:
I doit in a new PR after this one merged.